### PR TITLE
Feature/SM-1 Implemented the login story

### PR DESCRIPTION
**Pull request by IasminHunTen**

### CODE REVIEW

**Comment by Iasminimus** [comment]
Please resolve conflicts!

File: src/loose/oose/fis/lab/student/manager/sample.fxml [hunk 1]
```diff
@@ -3,6 +3,6 @@
```
Comment (Iasminimus) [inline src/loose/oose/fis/lab/student/manager/sample.fxml:3]: Remove unused file

File: src/loose/oose/fis/lab/student/manager/Controller.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package loose.oose.fis.lab.student.manager;
```
Comment (Iasminimus) [inline src/loose/oose/fis/lab/student/manager/Controller.java:1]: Remove unused file

File: src/loose/oose/fis/lab/student/manager/controllers/LoginController.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+            loginMessage.setText("Password cannot be empty");
+            return;
+        }
+
+        if (username.equals("student") && password.equals("student")) {
+            loginMessage.setText("Logged in as student!");
+            return;
+        }
+
+        if (username.equals("teacher") && password.equals("teacher")) {
```
Comment (Iasminimus) [inline src/loose/oose/fis/lab/student/manager/controllers/LoginController.java:37]: If login as a teacher succeeds, the user should be taken to the view-students scene

**Comment by IasminHunTen** [comment]
done, aprove my merge pls

**Pull request merged**